packages/consumption/src/modules/attributes/AttributesController.ts

```diff
@@ -536,11 +536,11 @@ export class AttributesController extends ConsumptionBaseController {
         });
 
         if (!predecessor.isRepositoryAttribute(this.identity.address)) {
-            return ValidationResult.error(CoreErrors.attributes.invalidPredecessor("Predecessor is not a valid repository attribute."));
+            return ValidationResult.error(CoreErrors.attributes.predecessorIsNotRepositoryAttribute());
         }
 
         if (!successor.isRepositoryAttribute(this.identity.address)) {
-            return ValidationResult.error(CoreErrors.attributes.invalidSuccessor("Successor is not a valid repository attribute."));
+            return ValidationResult.error(CoreErrors.attributes.successorNotValidRepositoryAttribute());
```

Comment:
Analogously to above, I recommend changing it to `successorIsNotRepositoryAttribute`.

Reply:
Is there not some validation happening regarding the successor? I'm with you regarding inserting `Is` though

Reply:
This error is returned, if the successor is not a RepositoryAttribute. Hence, calling the error `successorIsNotRepositoryAttribute` seems accurate to me.

Reply:
I looked again and have adopted your change, but also renamed `genericValidationError` into `successorIsNotAValidAttribute`